doc/api/crypto.md

```diff
@@ -3504,6 +3492,18 @@ the corresponding digest algorithm. This does not work for all signature
 algorithms, such as `'ecdsa-with-SHA256'`, so it is best to always use digest
 algorithm names.
 
+### `crypto.constants`
```

Comment:
`o` is before `r` in the latin alphabet, `crypto.constants` should appear before `crypto.createVerify`